src/main/java/com/joeyfrazee/nifi/reporting/ElasticsearchProvenanceReporter.java

```diff
@@ -35,14 +35,17 @@
 import org.apache.http.auth.AuthScope;
 import org.apache.http.auth.UsernamePasswordCredentials;
 import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.apache.nifi.annotation.configuration.DefaultSchedule;
 import org.apache.nifi.annotation.documentation.CapabilityDescription;
 import org.apache.nifi.annotation.documentation.Tags;
 import org.apache.nifi.components.PropertyDescriptor;
 import org.apache.nifi.processor.util.StandardValidators;
 import org.apache.nifi.reporting.ReportingContext;
+import org.apache.nifi.scheduling.SchedulingStrategy;
 import org.elasticsearch.client.RestClient;
 
 @Tags({"elasticsearch", "provenance"})
+@DefaultSchedule(strategy = SchedulingStrategy.TIMER_DRIVEN, period = "1 min")
```

Comment:
@cameron-thomson-bsl deserves a comment explaining why. I.e. explaining that ideally we'd use an env var to makes this configurable, but can't because.... Instead using a more sensible default of 1 min so that ....